templates/compositions/ec-component-content-language-switcher/content-language-switcher.html.twig

```diff
@@ -2,77 +2,101 @@
 
 {#
   Parameters:
+    - "unavailable_language" (string) (default: ''): the name of the unavailable language.
     - "expandable_id" (string) (default: ''): unique ID for the expandable component.
-    - "unavailable" (string) (default: ''): language currently unavailable.
-    - "current" (string) (default: ''): language currently displayed instead of the selected, unavailable, one.
-    - "options" (array) (default: []): array of links to the content in different languages. Format:
+    - "languages" (array) (default: []): array of links to the content in different languages. Format:
       [
         {
           "href" (string) URL of the content (default: ''),
           "hreflang" (string) language code to be used as link attribute,
-          "label" (string) link label, i.e. the language name in its native form (default: '')
+          "label" (string) link label, i.e. the language name in its native form (default: ''),
+          "current" (boolean) whether the language is the current page language or not (default: false)
         },
         ...
       ]
-    - "icon_path" (string) (default: ''): path to ECL SVG icon assets.
 #}
-
-<div class="ecl-lang-select-page">
-  <div class="ecl-u-d-flex ecl-u-align-items-start ecl-u-font-s">
-    <span class="ecl-lang-select-page__unavailable_language ecl-u-d-flex ecl-u-align-items-center ecl-u-pt-xs">
-      {% include '@ecl-twig/icon' with {
+  <div class="ecl-lang-select-page ecl-u-bg-grey-5 ecl-u-pv-m">
+    <div class="ecl-container">
+      {% include '@ecl-twig/message' with {
+        title: 'The @language language is unavailable.'|t({'@language': unavailable_language}),
+        variant: 'warning',
+        extra_classes: 'ecl-u-bg-transparent ecl-u-mb-m',
```

Comment:
In EU component library message looks a bit different and looks like related to `ecl-u-bg-transparent` class:
![image](https://user-images.githubusercontent.com/9432036/126977007-07878bc9-b0fd-4d34-8001-253e2eac74e0.png)

Reply:
True, I've fixed it.